src/main/webapp/data/userMapData.json

```diff
@@ -1104,13 +1132,12 @@
       "Creighton University",
       "Crooms Academy of Information Technology",
       "CSU Chico",
```

Comment:
Not your fault, but can help remove this entry? It's a duplicate of the existing entry `California State University, Chico` above.